CMakeLists.txt

```diff
@@ -180,4 +180,4 @@ endif()
         FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.pc"
         DESTINATION ${NLOHMANN_JSON_PKGCONFIG_INSTALL_DIR}
     )
-endif()
+endif() # JSON_Install
```

Comment:
Please remove the comment after the `endif()`.

Reply:
Done!